flask/app/templates/index.html

```diff
@@ -3,6 +3,22 @@
 {% block title %} Index {% endblock %}
 {% block content %}
 <div class="container">
-    <div>Index</div>
+    <h1>Rock! Paper! Scissors! Showdown </h1>
+    <div class="flex-container">
+        <span class="circle-dot">
+            <image src="./static/img/icons8-rock-80.png"></image>
```

Comment:
I would change the `<image>` to `<img>` tags, since img is the standard, and you also used img in your css so it would be better to be consistent